content/docs/other-guides/upgrade.md

```diff
@@ -12,31 +12,25 @@ deployments.
 
 Updating your deployment is a two step process:
 
-1. Update your ksonnet application:
+1. Update your kustomize manifests:
 
-  1. We recommend checking your app into source control to back it up before 
+  1. We recommend checking your local packages into source control to back it up before 
      proceeding.
-  1. Use the script 
-     [upgrade_ks_app.py](https://github.com/kubeflow/kubeflow/tree/{{< params "githubbranch" >}}/scripts)
-     to update your ksonnet app with the current version for the Kubeflow 
-     packages.
-
-     Note: ksonnet is working on support for this capability. 
-     See https://github.com/ksonnet/ksonnet/issues/237
+  1. Use `kfctl` to download the desired version of Kubeflow, e.g.:
+     ```
+     export KF_VER=v0.6.0
+     kfctl init ${KF_APP} --version=${KF_VER} --package-manager=kustomize@${KF_VER}
```

Comment:
Everywhere else we refer to the app directory as `KFAPP` rather than `KF_APP`. To help the reader understand, please use `KFAPP` here too. 

Also, export the variable before using it - many people get confused and assume the Kubeflow installation has created the variable for them.

```
export KFAPP="<the name of your Kubeflow application directory>"
````